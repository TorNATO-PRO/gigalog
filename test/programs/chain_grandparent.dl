parent(john, mary).
parent(mary, bob).

grandparent(X, Y) :- parent(X, Z), parent(Z, Y).
