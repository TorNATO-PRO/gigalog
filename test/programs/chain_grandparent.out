grandparent(john,bob)
parent(john,mary)
parent(mary,bob)
